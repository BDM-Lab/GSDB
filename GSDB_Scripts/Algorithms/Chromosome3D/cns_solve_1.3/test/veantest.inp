! CNS test of the VEAN energy term
! Alexandre Bonvin, February 2009
!
struc
data_cns_mtf

_cns_mtf.title
; FILENAME="veantest.psf"
  DATE:17-Feb-2009  17:34:01       created by user: abonvin
  VERSION:1.2
;

loop_
_cns_mtf_atom.id
_cns_mtf_atom.segment_id
_cns_mtf_atom.residue_id
_cns_mtf_atom.residue_name
_cns_mtf_atom.atom_name
_cns_mtf_atom.chemical_type
_cns_mtf_atom.charge
_cns_mtf_atom.atom_mass
1 'A' '8' 'VAL' 'N' 'NH1' -0.570000 14.0070
2 'A' '8' 'VAL' 'HN' 'H' 0.370000 1.00800
3 'A' '8' 'VAL' 'CA' 'CH1E' 0.200000 12.0110
4 'A' '8' 'VAL' 'CB' 'CH1E' 0.00000 12.0110
5 'A' '8' 'VAL' 'CG1' 'CH3E' 0.00000 12.0110
6 'A' '8' 'VAL' 'CG2' 'CH3E' 0.00000 12.0110
7 'A' '8' 'VAL' 'C' 'C' 0.500000 12.0110
8 'A' '8' 'VAL' 'O' 'O' -0.500000 15.9990
9 'A' '9' 'VAL' 'N' 'NH1' -0.570000 14.0070
10 'A' '9' 'VAL' 'HN' 'H' 0.370000 1.00800
11 'A' '9' 'VAL' 'CA' 'CH1E' 0.200000 12.0110
12 'A' '9' 'VAL' 'CB' 'CH1E' 0.00000 12.0110
13 'A' '9' 'VAL' 'CG1' 'CH3E' 0.00000 12.0110
14 'A' '9' 'VAL' 'CG2' 'CH3E' 0.00000 12.0110
15 'A' '9' 'VAL' 'C' 'C' 0.500000 12.0110
16 'A' '9' 'VAL' 'O' 'O' -0.500000 15.9990
17 'A' '10' 'GLY' 'N' 'NH1' -0.570000 14.0070
18 'A' '10' 'GLY' 'HN' 'H' 0.370000 1.00800
19 'A' '10' 'GLY' 'CA' 'CH2G' 0.200000 12.0110
20 'A' '10' 'GLY' 'C' 'C' 0.500000 12.0110
21 'A' '10' 'GLY' 'O' 'O' -0.500000 15.9990
-1 ' ' ' ' ' ' ' ' ' ' -1.00000 -1.00000

loop_
_cns_mtf_bond.id[1]
_cns_mtf_bond.id[2]
1 2
1 3
3 4
4 5
4 6
3 7
7 8
9 10
9 11
11 12
12 13
12 14
11 15
15 16
7 9
17 18
17 19
19 20
20 21
15 17
-1 -1

loop_
_cns_mtf_angle.id[1]
_cns_mtf_angle.id[2]
_cns_mtf_angle.id[3]
2 1 3
1 3 4
1 3 7
3 4 5
3 4 6
4 3 7
5 4 6
3 7 8
10 9 11
9 11 12
9 11 15
11 12 13
11 12 14
12 11 15
13 12 14
11 15 16
3 7 9
8 7 9
7 9 11
7 9 10
18 17 19
17 19 20
19 20 21
11 15 17
16 15 17
15 17 19
15 17 18
-1 -1 -1

loop_
_cns_mtf_dihedral.id[1]
_cns_mtf_dihedral.id[2]
_cns_mtf_dihedral.id[3]
_cns_mtf_dihedral.id[4]
5 4 3 1
13 12 11 9
7 9 11 15
7 9 11 15
7 9 11 15
7 9 11 15
7 9 11 15
7 9 11 15
7 9 11 12
7 9 11 12
7 9 11 12
7 9 11 12
7 9 11 12
7 9 11 12
4 3 7 9
4 3 7 9
4 3 7 9
4 3 7 9
4 3 7 9
4 3 7 9
4 3 7 8
4 3 7 8
4 3 7 8
4 3 7 8
4 3 7 8
4 3 7 8
15 17 19 20
15 17 19 20
15 17 19 20
15 17 19 20
15 17 19 20
15 17 19 20
12 11 15 17
12 11 15 17
12 11 15 17
12 11 15 17
12 11 15 17
12 11 15 17
12 11 15 16
12 11 15 16
12 11 15 16
12 11 15 16
12 11 15 16
12 11 15 16
-1 -1 -1 -1

loop_
_cns_mtf_improper.id[1]
_cns_mtf_improper.id[2]
_cns_mtf_improper.id[3]
_cns_mtf_improper.id[4]
7 3 9 8
9 7 11 10
3 7 9 11
15 11 17 16
17 15 19 18
11 15 17 19
-1 -1 -1 -1

loop_
_cns_mtf_explicit_nonbonded_exclusion.inb
-1

loop_
_cns_mtf_explicit_nonbonded_exclusion.iblo
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
-1

loop_
_cns_mtf_group_linked_list.first_atom_id
0
8
16
-1
end

coor
ATOM      1  N   VAL     8       3.242   2.738  -8.765  1.00 15.00      A   
ATOM      2  HN  VAL     8       2.412   2.217  -8.769  1.00 15.00      A   
ATOM      3  CA  VAL     8       3.497   3.671  -9.844  1.00 15.00      A   
ATOM      4  CB  VAL     8       2.257   4.271 -10.493  1.00 15.00      A   
ATOM      5  CG1 VAL     8       0.978   4.010  -9.708  1.00 15.00      A   
ATOM      6  CG2 VAL     8       2.377   5.667 -11.066  1.00 15.00      A   
ATOM      7  C   VAL     8       4.785   4.469  -9.663  1.00 15.00      A   
ATOM      8  O   VAL     8       5.022   5.061  -8.588  1.00 15.00      A   
ATOM      9  N   VAL     9       5.680   4.345 -10.650  1.00 15.00      A   
ATOM     10  HN  VAL     9       5.438   3.799 -11.427  1.00 15.00      A   
ATOM     11  CA  VAL     9       7.012   4.983 -10.636  1.00 15.00      A   
ATOM     12  CB  VAL     9       8.141   3.951 -10.657  1.00 15.00      A   
ATOM     13  CG1 VAL     9       7.811   2.873  -9.640  1.00 15.00      A   
ATOM     14  CG2 VAL     9       8.163   3.271 -12.025  1.00 15.00      A   
ATOM     15  C   VAL     9       7.210   6.086 -11.696  1.00 15.00      A   
ATOM     16  O   VAL     9       6.385   6.302 -12.588  1.00 15.00      A   
ATOM     17  N   GLY    10       8.308   6.832 -11.613  1.00 15.00      A   
ATOM     18  HN  GLY    10       8.983   6.657 -10.924  1.00 15.00      A   
ATOM     19  CA  GLY    10       8.473   7.914 -12.576  1.00 15.00      A   
ATOM     20  C   GLY    10       9.054   9.188 -11.954  1.00 15.00      A   
ATOM     21  O   GLY    10       9.034   9.477 -10.762  1.00 15.00      A   
end

vean
  class vea1
  force 40.0 10.0
  assign (resid 8 and name N)(resid 8 and name HN)(resid 10 and name N)(resid 10 and name HN) 33.3 19.8 146.7 19.8 
  assign (resid 9 and name N)(resid 9 and name HN)(resid 10 and name N)(resid 10 and name HN) 31.6 19.1 148.4 19.1
end

flag excl * incl vean end

energy end

vean print threshold=5.0 class vea1 end

stop

