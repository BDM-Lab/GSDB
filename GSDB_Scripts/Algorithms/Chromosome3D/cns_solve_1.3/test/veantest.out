          ============================================================
          |                                                          |
          |            Crystallography & NMR System (CNS)            |
          |                         CNSsolve                         |
          |                                                          |
          ============================================================
           Version: 1.3
           Status: General release
          ============================================================
           Written by: A.T.Brunger, P.D.Adams, G.M.Clore, W.L.DeLano,
                       P.Gros, R.W.Grosse-Kunstleve,J.-S.Jiang,J.M.Krahn,
                       J.Kuszewski, M.Nilges, N.S.Pannu, R.J.Read,
                       L.M.Rice, G.F.Schroeder, T.Simonson, G.L.Warren.
           Copyright (c) 1997-2010 Yale University
          ============================================================
           Running on machine: lewis4-r710-login-node223 (x86_64/Linux,64-bit)
                               with 16 threads
           Program started by: oeow39
           Program started at: 16:08:22 on 05-Jul-2018
          ============================================================

 FFT3C: Using FFTPACK4.1

 CNSsolve>! CNS test of the VEAN energy term 
 CNSsolve>! Alexandre Bonvin, February 2009 
 CNSsolve>! 
 CNSsolve>struc 
 STRUcture>data_cns_mtf 
 REMARKS FILENAME="veantest.psf"
 REMARKS DATE:17-Feb-2009  17:34:01       created by user: abonvin
 REMARKS VERSION:1.2
 Status of internal molecular topology database:
 -> NATOM=         21(MAXA=     1000000)  NBOND=         20(MAXB=     1000000)
 -> NTHETA=        27(MAXT=     2000000)  NGRP=           3(MAXGRP=   1000000)
 -> NPHI=          44(MAXP=     2000000)  NIMPHI=         6(MAXIMP=   1000000)
 -> NNB=            0(MAXNB=    1000000) 
 STRUcture>end 
 CNSsolve> 
 CNSsolve>coor 
 COOR>ATOM      1  N   VAL     8       3.242   2.738  -8.765  1.00 15.00      A 
 COOR>ATOM      2  HN  VAL     8       2.412   2.217  -8.769  1.00 15.00      A 
 CNSsolve> 
 CNSsolve>vean 
 VEANORANGLES>  class vea1 
 VEANORANGLES>  force 40.0 10.0 
Setting force consts for class VEA1     to   40.000  10.000
 VEANORANGLES>  assign (resid 8 and name N)(resid 8 and name HN)(resid 10 and name N)(resid 10 and name HN) 33.3 19.8 146.7 19.8 
 SELRPN:      1 atoms have been selected out of     21
 SELRPN:      1 atoms have been selected out of     21
 SELRPN:      1 atoms have been selected out of     21
 SELRPN:      1 atoms have been selected out of     21
 VEANORANGLES>  assign (resid 9 and name N)(resid 9 and name HN)(resid 10 and name N)(resid 10 and name HN) 31.6 19.1 148.4 19.1 
 SELRPN:      1 atoms have been selected out of     21
 SELRPN:      1 atoms have been selected out of     21
 SELRPN:      1 atoms have been selected out of     21
 SELRPN:      1 atoms have been selected out of     21
 VEANORANGLES>end 
 CNSsolve> 
 CNSsolve>flag excl * incl vean end 
 CNSsolve> 
 CNSsolve>energy end 
 --------------- cycle=     1 --------------------------------------------------
 | Etotal =0.984      grad(E)=3.764      E(VEAN)=0.984                         |
 -------------------------------------------------------------------------------
 CNSsolve> 
 CNSsolve>vean print threshold=5.0 class vea1 end 
The following vectorangles have delta
greater than the cutoff:
(calculated)(observed borderA borderB borderC borderD)(delta)
class VEA1    
          A    8    VAL  N    A    8    VAL  HN   A    10   GLY  N    A    10   GLY  HN   119.416  13.500  53.100 126.900 166.500   7.484
  RMS diff. =   5.300, #(violat.> 5.00)=    1. of      2 vectorangles
 CNSsolve> 
 CNSsolve>stop 
          ============================================================
           Maximum dynamic memory allocation:     8201488 bytes
           Maximum dynamic memory overhead:           208 bytes
           Program started at: 16:08:22 on 05-Jul-2018
           Program stopped at: 16:08:22 on 05-Jul-2018
           CPU time used:       0.0063 seconds
          ============================================================
